configure.py

```diff
@@ -1496,17 +1503,43 @@ def _generate_target_source_files_isa_properties(fd, target):
     @staticmethod
     def _write_header(fd):
         fd.write('cmake_minimum_required(VERSION 2.8.0)\n')
-        fd.write('project(botan LANGUAGES CXX)\n\n')
+        fd.write('project(botan)\n\n')
+        fd.write('if(POLICY CMP0042)\n\n')
+        fd.write('cmake_policy(SET CMP0042 NEW)\n\n')
         fd.write('cmake_policy(SET CMP0042 NEW)\n\n')
+        fd.write('endif()\n\n')
+        fd.write('function(to_cmake_paths paths results)\n')
+        fd.write('    set(_results)\n')
+        fd.write('	foreach(path ${paths})\n')
```

Comment:
If we still need this function, let's use 4 space indent for all the function body